agents/influxdb_publisher/influxdb_publisher.py

```diff
@@ -90,6 +90,8 @@ def process_incoming_data(self):
         """
         while not self.incoming_data.empty():
             data, feed = self.incoming_data.get()
+            if feed['agg_params'].get('exclude_influx', False):
+                return
```

Comment:
Should this should be "continue", not "return"?  You want to empty the queue on this call and not rely on call rate > block arrival rate.  (I might have read the code too quickly, but please check.)

Reply:
Good catch, thanks! Should definitely be continue in both cases.